+siibra/atlas.m

```diff
@@ -0,0 +1,8 @@
+function atlas = atlas(atlas_name)
+% Construct atlas by name
+%   Detailed explanation goes here
+    atlases = siibra.internal.initAtlases(false);
+    atlas = atlases(siibra.internal.fuzzyMatching(atlas_name, {atlases.name}));
```

Comment:
Recommend to use a string array type for the atlas object `Name` property.

This avoids needing to use cell arrays, which are best reserved for heterogeneous type arrays/lists.